web-wallet/src/routes/(app)/settings/__tests__/__snapshots__/page.spec.js.snap

```diff
@@ -161,8 +161,8 @@ exports[`Settings > Resetting the wallet > should show an error if clearing loca
           <p
             class="s-EBshooXNO1eQ"
           >
-            These functionalities are currently disabled in the beta
-					version of the web wallet and will be enabled in a future update.
+            These functionalities are currently disabled in the beta version of
```

Comment:
I will create an issue to extract this as a constant that we can reuse here – as we have this message in 3 places.